libs/agora/gene-comparison-tool/src/lib/gene-comparison-tool.component.html

```diff
@@ -9,43 +9,30 @@
       <div class="gct-header-inner">
         <div class="gct-header-left">
           <p-button
+            label="Filter Genes"
             (click)="filterPanel.toggle()"
+            [rounded]="true"
+            icon="pi pi-filter"
             size="small"
-            rounded="true"
             pTooltip="Filter genes by Nominations, Teams, Cohort, and more"
             tooltipPosition="right"
             tooltipStyleClass="tooltip"
-          >
-            <svg
-              width="14"
-              height="14"
-              viewBox="0 0 14 14"
-              version="1.1"
-              xmlns="http://www.w3.org/2000/svg"
-            >
-              <path
-                fill="currentColor"
-                d="M0,0.7C0,0.3,0.3,0,0.7,0h12.5C13.7,0,14,0.3,14,0.7v2.3c0,0.2-0.1,0.4-0.2,0.5L9,8.2v4.4 c0,0.3-0.2,0.6-0.6,0.7L5.9,14c-0.2,0.1-0.4,0-0.6-0.1c-0.2-0.1-0.3-0.3-0.3-0.6v-5L0.2,3.7C0.1,3.5,0,3.3,0,3.1V0.7z"
-              />
-            </svg>
-            Filter Genes
-          </p-button>
```

Comment:
SVG code no longer needed since we can simply use a filter icon on PrimeNG button